src/configure/configure.ts

```diff
@@ -124,46 +187,21 @@ class Orchestrator {
             await this.getAzureSession();
             await this.getGithubPatToken();
             await this.getSelectedPipeline();
+            if (!resourceNode) {
+                let selectedResource = await this.getAzureResource();
```

Comment:
I will Let Kanika reply here, but as per my understanding getAzureresource() internally takes care of that by throwing `Messages.setupAlreadyConfigured`

Reply:
That's handled in WebAppResourceSelector.ts
```
     if (await appServiceClient.isScmTypeSet((<GenericResource>selectedResource.data).id)) {
               await openBrowseExperience((<GenericResource>selectedResource.data).id);
               throw new Error(Messages.setupAlreadyConfigured);
          }
```

Reply:
you are catching the exception in this.getAzureResource() and setting `this.continueOrchestration = false` but this variable is not checked here so it will continue to call selectTemplate() and then getAllPipelineTemplateInputs()

Reply:
added a check for ``` this.continueOrchestration ```